test/support/test_helpers.ex

```diff
@@ -6,9 +6,11 @@ defmodule Benchee.TestHelpers do
   # retry tests that are doing actual benchmarking and are flaky
   # on overloaded and/or slower systems
   def retrying(asserting_function, n \\ @default_retries)
+
   def retrying(asserting_function, 1) do
     asserting_function.()
   end
+
```

Comment:
I knew this was coming. I thought I might be ready, clearly I'm not. For me this is the worst one... I loved seeing which functions belong to each other.